index.html

```diff
@@ -3,8 +3,170 @@
+                          <i class="fab fa-pinterest-square"></i>
+                          <i class="fab fa-google-plus-square"></i>
+                      </div>
+                  </div>
+                </header>
+                <header class="nav-bottom ">
+                      <div class="continer">
+                        <div class="logo FL">
+                          <img src="image/logo.png" alt="LOGO">
+                        </div>
+
+                        <div class=" navg FL">
+                          <a class="nav-link nav-co" href="#">Home</a>
+                          <div class="dropdown DIB">
+                              <ul class="DIB">
+                                <li class="d-down DIB nav-co">Page <i class="fas fa-sort-down"></i>
+                                  <ul class="d-item">
+                                    <li class="nav-li nav-co">item one</li>
+                                    <li class="nav-li nav-co">item two</li>
+                                    <li class="nav-li nav-co">itam three</li>
+                                  </ul>
+                                </li>
+
+                                <li class="d-down DIB nav-co">Event <i class="fas fa-sort-down"></i>
+                                  <ul class="d-item">
+                                    <li class="nav-li nav-co">item one</li>
+                                    <li class="nav-li nav-co">item two</li>
+                                    <li class="nav-li nav-co">itam three</li>
+                                  </ul>
+                                </li>
+
+                                <li class="d-down DIB nav-co">Blog <i class="fas fa-sort-down"></i>
+                                  <ul class="d-item">
+                                    <li class="nav-li nav-co">item one</li>
+                                    <li class="nav-li nav-co">item two</li>
+                                    <li class="nav-li nav-co">itam three</li>
+                                  </ul>
+                                </li>
+
+                                <li class="d-down DIB nav-co">Shop <i class="fas fa-sort-down"></i>
+                                  <ul class="d-item">
+                                    <li class="nav-li nav-co">item one</li>
+                                    <li class="nav-li nav-co">item two</li>
+                                    <li class="nav-li nav-co">itam three</li>
+                                  </ul>
+                                </li>
+
+                              </ul>
+                              <a class="nav-link nav-co" href="#">Content</a>
+                          </div>
+                        </div>
+                    </div>
+                      <div class="clear"></div>
+
+                </header>
+            </header>
+        </header>
+    </div>
+    <div class="sidebar">
+      <div class="slider">
```

Comment:
fix the image 
and add a dot for the slider 
why don't use plugins it's easy for you?